sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/StorageBlobClient.java

```diff
@@ -392,28 +399,28 @@ public ServiceCall downloadWithHeaders(String containerName,
             callback);
     }
 
-    public void stageBlock(String containerName,
+    public BlockBlobsStageBlockResponse stageBlock(String containerName,
```

Comment:
The methods in this category are those with "simplest" signature, which should not return any `Response` type but instead returns the actual value in the response, in this case it should be `void`.

Reply:
You are right, I must have missed this for `stageBlock`.